client/state/selectors/is-plugin-replacing-wp-admin-editor.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import isPluginActive from 'state/selectors/is-plugin-active';
+
+// List of known plugins that replaces the default WP Admin editor.
+const pluginBlacklist = [ 'elementor' ];
+
+/**
+ * Indicates whether a site has active a plugin that replaces the default WP Admin editor with other custom editor.
+ *
+ * @param {Object} state - Global state tree
+ * @param {Number} siteId - Site ID
+ * @return {Boolean} - truthiness of a site having at least one conflicting plugin active
+ */
+export default function isPluginReplacingWpAdminEditor( state, siteId ) {
```

Comment:
How do you feel about `hasEditorConflictingPlugin`?

Reply:
Sounds good, but I think we should indicate clearly we're talking about the WP Admin editor and not about the Calypso editor, so `hasWpAdminEditorConflictingPlugin ` should clarify that.